packages/core/strapi/bin/strapi.js

```diff
@@ -322,6 +322,11 @@ program
           }
           thisCommand.opts().fromToken = answers.fromToken;
         }
+
+        await confirmMessage(
+          'The transfer will delete all data in the local database and media files. Are you sure you want to proceed?',
```

Comment:
```suggestion
          'The transfer will delete all data in the database and local. Are you sure you want to proceed?',
```

Just a small suggestion, because:
1) the database doesn't have to be local IMO
2) the media library handles all kind of assets - not only media files

:)

Reply:
I've updated both messages, does it look good now?